src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs

```diff
@@ -1,339 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Please leave these infrastructure pieces so that I can do https://github.com/aspnet/AspNetCore/issues/5485.

Reply:
That should get resolved as part of https://github.com/aspnet/AspNetCore/pull/17410, Is there something specifically in that issue that requires these types?

Reply:
Spoke offline, this looks ok